File: patterns/c-patterns/pattern76.c
// Pattern 76: ...

// * * * * *
//   *     *
//     *   *
//       * *
//         *

#include <stdio.h>

int main()
{
    int n = 5;
    int i, j;

    for (i = n; i >= 1; i--)
    {
        for (j = 1; j <= n; j++)
        {
            if (j == n - i + 1 || j == n || i == n)
            {
                printf("* ");
            }
            else
            {
                printf("  ");
            }
        }
        printf("\n");
    }
    return 0;
}

File: patterns/python-patterns/pattern76.py
# Pattern 76: ...

# * * * * *
#   *     *
#     *   *
#       * *
#         *

n = 5
for i in range(n, 0, -1):
    for j in range(1, n + 1):
        if j == n - i + 1 or j == n or i == n:
            print("* ", end="")
        else:
            print("  ", end="")
    print()
